src/rp2/plugin/report/us/open_positions.py

```diff
@@ -120,9 +122,9 @@ def generate(
             unit_cost_basis: RP2Decimal
             unit_cost_basis = net_cost / tot_crypto_bal
             if unit_cost_basis >= _FIAT_UNIT_WINDOW_2_DECIMAL:
-                unit_cost_basis = RP2Decimal(round(unit_cost_basis, 2))
+                unit_cost_basis = RP2Decimal(unit_cost_basis).quantize(RP2Decimal("1.00"))
```

Comment:
mypy does complain about this, regardless of if I use Decimal or RP2Decimal inside of the quantize. Seems to work, just not sure if there's a better way to express this to make mypy happy.
```
src\rp2\plugin\report\us\open_positions.py:125:35: error: Incompatible types in assignment (expression has type "Decimal", variable has type "RP2Decimal")  [assignment]
                    unit_cost_basis = RP2Decimal(unit_cost_basis).quantize(RP2Decimal("1.00"))
                                      ^
```

Reply:
It should be:
```suggestion
                unit_cost_basis = unit_cost_basis.quantize(FIAT_DECIMAL_MASK)
```
You'll also need to import FIAT_DECIMAL_MASK from rp2.rp2_decimal

Reply:
I had to tweak this a little. First, since I'm using 2 and 4 decimal output, I decided not to use the define from rp2_decimal for clarity, but that pointer was useful for taking a 2nd fresh look at the RP2Decimal class. I noticed quantize is used a test there, but as an assignment, mypy complains about assigning a decimal result to an RP2Decimal variable so the second tweak to the above was to init the statement result to a new RP2Decimal object. Works correctly and satisfies mypy!